### PR TITLE
Add roles for backend-setup and firewall configuration

### PR DESCRIPTION
**Pull request by sac**
* Create sub-roles backend-setup and firewall_config
* Based on the variables call playbooks to create thick/thin lv

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,176 @@
...
+
+
 Example Playbook
 ----------------
 
-```yaml
+Configure the ports and services related to GlusterFS, create logical volumes and mount them.
+
+
+```
```
Comment (devyanikota) [inline README.md]: nit: replace with  ```yaml
it'll help highlight yaml format.

File: examples/setup_vg.retry [hunk 2]
```diff
@@ -0,0 +1 @@
+10.70.43.52
```
Comment (devyanikota) [inline examples/setup_vg.retry]: This file can be removed :)

File: roles/backend_reset/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+---
+# tasks file for backend-reset
```
Comment (devyanikota) [inline roles/backend_reset/tasks/main.yml]: Empty tasks file?

File: roles/backend_setup/README.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+```
+
+
+Example Playbook
+----------------
+
+Configure the ports and services related to GlusterFS, create logical volumes and mount them.
+
+
+```
```
Comment (devyanikota) [inline roles/backend_setup/README.md]: Adding ```yaml will help highlight the yaml syntax

Comment (sac) [reply roles/backend_setup/README.md]: Same as above, it is not actually needed.

File: examples/cache_setup.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  hosts: vdos
+  gather_facts: false
+
+  # gluster-ansible -i inventory_file cache_setup.yml --tags cachesetup
+
+  vars:
+     # Variables for creating volume group
+     glusterfs_infra_vg_name: glusterfs_vg
+
+     # Note: The _pvs varialbe is not needed when cachesetup is done along with
```
[the remark this answers is not in the document]
Comment (devyanikota) [reply examples/cache_setup.yml]: @sac I meant there is a typo :)

**Pull request merged**